app/src/components/Map/AreasList.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'preact';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { COLORS } from 'constants';
+import areasPanelStyles from 'styles/components/map/areas-panel.scss';
+import controlPanelStyles from 'styles/components/control_panel.scss';
+import buttonStyles from 'styles/components/map/button.scss';
+import Toggle from 'components/Shared/Toggle';
+
+function AreasList({ areas, recentlyCreated, toggleAreaVisibility, deleteArea }) {
```

Comment:
Please create a class that extends the Component class instead, unless you are really sure these components won't grow